README.md

```diff
@@ -273,6 +273,51 @@ make uninstall-mesh
 
 ## Contributing
 
+### Create a new version
+
+To create a new version of the serverless-operator (usually after a release branch has
+been cut), there are a few steps we have to do. These steps can be done one after the
+other and do not have to be sent as one PR, to avoid clogging up the respective PR.
+
+#### 1. Update the version metadata of the serverless-operator
+
+The first thing we usually do is to update the version metadata of the operator. That
+mostly includes bumping the version to the next desired version (i.e. 1.12 -> 1.13).
+This is done by adjusting the respective settings in the `project` and `olm` part of
+[`project.yaml`](./olm-catalog/serverless-operator/project.yaml). The settings to be
+changed usually are `project.version`, `olm.replaces` and `olm.skipRange`.
```

Comment:
what about the `registry.svc.ci.openshift.org/openshift/openshift-serverless-UPDATED-1).0:serverless-bundle` in the `hack/lib/catalogsource.bash` ?

Reply:
Good one, forgot that.

Reply:
@markusthoemmes the issue here, with the `UPDATED-1` is, that it does require an actual _first_ production build from Warren, of the cut release branch, AFAIK ...